libbeat/docs/command-reference.asciidoc

```diff
@@ -113,6 +113,30 @@ template.
 Exports the current configuration to stdout. If you use the `-c` flag, this
 command exports the configuration that's defined in the specified file.
 
+
+
+*`dashboard`*::
+Exporting a dashboard allows to store a dashboard on disk in a
+module and load it automatically. The following command can be used:
++
+["source","shell",subs="attributes"]
+----
+$ {beatname_lc} export dashboard --id="dashboard-id" > dashboard.json
```

Comment:
We don't show $ in the other examples (makes copy/pasting harder).

Reply:
makes sense. I wonder why I have it here. Pretty sure i copied from somewhere.